lib/fetch/body.js

```diff
@@ -397,7 +398,47 @@ function bodyMixinMethods (instance) {
 
       // If mimeType’s essence is "multipart/form-data", then:
       if (/multipart\/form-data/.test(contentType)) {
-        throw new NotSupportedError('multipart/form-data not supported')
+        const headers = {}
+        for (const [key, value] of this.headers) headers[key.toLowerCase()] = value
+
+        const responseFormData = new FormData()
+
+        let busboy
```

Comment:
since we throw when failing to resolve anyway, maybe we should move entire try-catch block into an internal function to make busboy const?

Reply:
Unless that is critical behavior to have, I would say this is a good place for MVP to iterate from and meets the initial criteria discussed with @mcollina

Reply:
Agreed